Doc/whatsnew/3.10.rst

```diff
@@ -366,6 +366,10 @@ New Features
 * Added :c:func:`PyUnicode_AsUTF8AndSize` to the limited C API.
   (Contributed by Alex Gaynor in :issue:`41784`.)
 
+* The :c:func:`PyType_FromModuleAndSpec` function now accepts NULL ``tp_doc``
+  slot.
```

Comment:
Please copy this text to the NEWS entry.

Reply:
done.